docs/xks/developer-guide/gitops.md

```diff
@@ -0,0 +1,71 @@
+---
+id: gitops
+title: GitOps a la XKS
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+## What is GitOps?
+
+> GitOps works by using Git as a single source of truth for declarative infrastructure and applications. With GitOps, the use of software agents can alert on any divergence between Git with what's running in [an environment], and if there's a difference, Kubernetes reconcilers automatically update or rollback the cluster depending on the case. &dash; _[Weave Works - Guide To GitOps](https://www.weave.works/technologies/gitops/)_
+
+XKS supports GitHub and Azure DevOps with almost identical workflows. XKF refers to as Git providers. For simplicity, we refer to their CI/CD automation as "pipelines". If you are using GitHub, whenever this text refers to "pipeline", think "GitHub Actions workflow". As you saw in the previous section, XKS comes with a set of pipelines that automatically detects app releases and promotes them through a series of environments. The allows both rapid iteration and strong validation of apps.
+
+XKS is built around [trunk-based development](https://trunkbaseddevelopment.com/).
+
+## Emilia updates an app
+
+In the previous section, we looked at deploying our first app to Kubernetes using a fully automatic flow. But what actually happened in that flow? This section tells the story about a developer called Emilia. She has updated an app packaged as a container image. A pipeline in the app's repository has just tagged the container image as `a8b91c33` and uploaded it to a container registry.
+
+The GitOps repository for Emilia's app has a `gitops-promotion.yaml` that looks like this (for much more details, see the [gitops-promotion](https://github.com/XenitAB/gitops-promotion) readme):
+
+```yaml
+prflow: per-app
+environments:
+  - env: dev
+    auto: true
+  - env: qa
+    auto: true
+  - env: prod
+    auto: false
+```
+
+The `dev` and `qa` environemnts have `auto: false` which means that new releases will be automatically applied, while `prod` environment is configured with `auto: false`. This means that pull requests must be merged by a human, presumably one that has verified that the update worked as expected in previous environments. The GitOps repository is configured to require checks on pull requests to pass in order to allow merge.
```

Comment:
`dev` and `qa` environemnts have `auto: true` (note that they are set to true, not false)

Reply:
Fixed.